clusterman/cli/toggle.py

```diff
@@ -22,20 +23,37 @@
 from clusterman.args import add_scheduler_arg
 from clusterman.args import subparser
 from clusterman.aws.client import dynamodb
+from clusterman.aws.client import sts
 from clusterman.cli.util import timeout_wrapper
 from clusterman.util import AUTOSCALER_PAUSED
 from clusterman.util import autoscaling_is_paused
 from clusterman.util import CLUSTERMAN_STATE_TABLE
 from clusterman.util import parse_time_string
 
+logger = colorlog.getLogger(__name__)
+
+
+@timeout_wrapper
+def check_account_id(cluster) -> None:
+    current_account_id = sts.get_caller_identity()
+    cluster_account_id = staticconf.read_string(f'clusters.{cluster}.aws_account_number')
+
+    if(current_account_id != cluster_account_id):
+        logger.warning(
```

Comment:
i would raise an error instead of logging here, because disabling or enabling the cluster using wrong iam user is incorrect operation so we shouldn't even attempt doing it

if we do decide to raise an error, let's also rename the function from `check_...` to `ensure_...`